addons/l10n_es/data/account_tax_data.xml

```diff
@@ -5071,6 +5071,29 @@
         <field name="amount" eval="-19"/>
         <field name="amount_type">percent</field>
         <field name="tax_group_id" ref="tax_group_retenciones_19"/>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'base',
+            }),
+
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'tax',
+                'account_id': ref('l10n_es.account_common_4751'),
```

Comment:
@pedrobaeza If it is like before we should remove this account.  Any idea if 4751 is better?

Reply:
Not sure to understand you, but the other tax coming from has also this tax account: https://github.com/odoo/odoo/pull/163986/files#diff-10a36ce61ec690213e5a4e29af69c30cd87e78fcfe79975bc6762161f8fa2a86R5046

Reply:
@pedrobaeza Indeed, but this is different than putting no account at all.

Reply:
Yeah, but the account should be there. Not having tax tags doesn't mean the tax amount should go to the expense account. It should go to this specific account. The question is that the tax model where this is reported doesn't exist yet in enterprise reports.